README.md

```diff
@@ -107,20 +106,53 @@ To use `tw` specific CLI options such as `--insecure`, use the `--cli=` flag, fo
 
 For example:
 
-```
+```bash
 seqerakit file.yaml --cli="--insecure"
 ```
 
 To use an SSL certificate that is not accepted by the default Java certificate authorities and specify a custom `cacerts` store as accepted by the `tw` CLI, you can specify the `-Djavax.net.ssl.trustStore=/absolute/path/to/cacerts` option enclosed in double quotes to `seqerakit` as you would to `tw`, preceded by `--cli=`.
 
 For example:
 
-```
+```bash
 seqerakit hello-world-config.yml --cli="-Djavax.net.ssl.trustStore=/absolute/path/to/cacerts"
 ```
 
 <b>Note</b>: Use of `--verbose` option for the `tw` CLI is currently not supported by `seqerakit`. Supplying `--cli="--verbose"` will raise an error.
 
+## YAML Configuration Options
+
+There are several options that can be provided in your YAML configuration file, that are handled specially by seqerakit and not `tw` specific CLI options.
+
+### 1. Pipeline parameters using `params` and `params-file`
+
+To specify pipeline parameters, you may either use `params:` to specify a list of parameters, or use `params-file:` to point to a parameters file.
+
+For example, to specify pipeline parameters within your YAML:
+
+```yaml
+params:
+  outdir: 's3://path/to/outdir'
+  fasta: 's3://path/to/reference.fasta'
+```
+
+Alternatively, to specify a file containing pipeline parameters:
+
+```yaml
+params-file: '/path/to/my/parameters.yaml'
+```
+
+Optionally, you may provide both:
+
+```yaml
+params-file: '/path/to/my/parameters.yaml'
+params:
+  outdir: 's3://path/to/outdir'
+  fasta: 's3://path/to/reference.fasta'
+```
+
+If duplicate parameters are provided, the parameters in `params-file` will take precedence.
```

Comment:
This feels the wrong way round to me, I'd have expected params specified here to take precedence over the file.

Reply:
I buy this, I don't feel strongly about this but I can see how `params:` overriding the file would mirror how we specify parameters on Tower.